modules/gdscript/gdscript.cpp

```diff
@@ -1994,8 +1994,9 @@ void GDScriptLanguage::get_reserved_words(List<String> *p_words) const {
 		"preload",
 		"signal",
 		"super",
-		"trait",
-		"yield",
+		"trait", // Not implemented yet, but reserved for future use. Highlight as a keyword to make errors easier to understand.
+		"namespace", // Not implemented yet, but reserved for future use. Highlight as a keyword to make errors easier to understand.
+		"yield", // Not implemented yet, but reserved for future use. Highlight as a keyword to make errors easier to understand.
```

Comment:
Can they be moved at the end of the array (before `nullptr`) with a comment that encompasses all three?
That would be nicer than duplicating a long comment three times.
```suggestion
		// The following keywords are not implemented currently, but reserved for (potential) future use.
		// We highlight them as keywords to make errors easier to understand.
		"trait", 
		"namespace",
		"yield",
```